homeassistant_cli/remote.py

```diff
@@ -64,6 +64,8 @@ def restapi(
 
     if ctx.token:
         headers["Authorization"] = "Bearer {}".format(ctx.token)
+    if ctx.password is not None:
+        headers["x-ha-access"] = ctx.password
```

Comment:
adding both headers will never make sense - but it also don't really hurt so let's just keep it simple.

Reply:
Is there maybe some way to tell click to output an error if both, token and password, are provided?

Reply:
It seems there is none but we can easily implement our own: https://github.com/pallets/click/issues/257. Should I do that?

Reply:
I think that would be great, but i'm ok if you do that as a separate pr since its mainly a nice to have.